README.md

```diff
@@ -75,9 +87,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1.inline styles
```

Comment:
You know, this isn't the answer we were going for, but you're sort-of correct: inline styles override IDs, which override classes. I'll give it to you!

We were going for the places you put CSS in general:

1. Inline styles `<div style="color:red">`
2. The `<style>` tag inside the `<head>`
3. External styles using `<link>`